Providers/SelectedListManager.tscn

```diff
@@ -7,6 +7,7 @@ script/source = "extends Node
 var selected_list = []
 signal selection_changed
 signal set_revealed_hidden
+signal closed_detailspanel
```

Comment:
This shouldn't be done by SelectedListManager. It's the details panel that should listen to SelectedListManager selection_changed signal and close itself when selected_list is empty